app/phpunit.xml.dist

```diff
@@ -6,6 +6,7 @@
          backupGlobals="false"
          colors="true"
          bootstrap="../autoload.php"
+         stderr="true"
```

Comment:
This makes sure \SpoonCookie and some other stuff setting headers doesn't break our scripts. PHPUnit breaks by default on setcookie, header and some other methods.